test/language/expressions/optional-chaining/runtime-semantics-evaluation.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Google, Inc.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: prod-OptionalExpression
+description: >
+  accessing optional value on undefined or null returns undefined.
+info: |
+  If baseValue is undefined or null, then
+    Return undefined.
+features: [optional-chaining]
+---*/
+
+const nul = null;
+const undf = undefined;
+assert.sameValue(undefined, nul?.a);
```

Comment:
for a follow up:

```js
var x = 0;
var y = null?.[(function() { x += 1 }())];

// assert value of x to verify if the `[Expression]` has got evaluated
// assert y is undefined.
```

Reply:
oh! you did that in `test/language/expressions/optional-chaining/short-circuiting.js`, nice!